packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -97,7 +97,7 @@
     </SF.Item>
   </Shw::Flex>
 
-  <Shw::Text::H4>With icon</Shw::Text::H4>
+  <Shw::Text::H3>With icon</Shw::Text::H3>
```

Comment:
@MelSumner probably better to pass the `@tag` argument to the `<hx>` level you want, without changing its visual appearance. Something like this:
```
<Shw::Text::H4 @tag="h3">With icon</Shw::Text::H4>
```

Reply:
If that's the visual appearance we want, why wouldn't we make the h3 look that way? Just curious.

Reply:
In most of the contexts using the normal sequence `Shw::Text::H1 > Shw::Text::H2 > Shw::Text::H3 > Shw::Text::H4` works well, visually. But in this specific context (I assume) the person that worked on the page wanted to reduce the visual weight of the text in relation to the other elements around.